### PR TITLE
RSE-1063 QAF Migration: 3 tests

### PR DESCRIPTION
**Pull request by DarwisNarvaezDev**
## :warning: This PR is to continue the work of this pr: https://github.com/rundeck/rundeck/pull/9011 by an involuntary error, was emptied and closed :warning:

### CODE REVIEW

File: functional-test/src/test/groovy/org/rundeck/util/gui/pages/execution/ExecutionShowPage.groovy [hunk 1]
```diff
@@ -24,6 +25,8 @@ class ExecutionShowPage extends BasePage {
     By viewContentOutputBy = By.cssSelector("#output")
     By viewButtonOutputBy = By.id("btn_view_output")
     By logOutputBy = By.className("execution-log__content-text")
+    By logOutputSwitchBy = By.id("btn_view_output")
```
Comment (ronaveva) [inline functional-test/src/test/groovy/org/rundeck/util/gui/pages/execution/ExecutionShowPage.groovy:28]: this locator already exist, line 26

File: functional-test/src/test/groovy/org/rundeck/util/gui/pages/execution/ExecutionShowPage.groovy [hunk 2]
```diff
@@ -7,6 +7,7 @@ import org.openqa.selenium.support.ui.ExpectedConditions
 import org.openqa.selenium.support.ui.WebDriverWait
 import org.rundeck.util.container.SeleniumContext
 import org.rundeck.util.gui.pages.BasePage
+
```
Comment (ronaveva) [inline functional-test/src/test/groovy/org/rundeck/util/gui/pages/execution/ExecutionShowPage.groovy:10]: no need for a space here

File: functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobTabsSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        jobShowPage.editJobLink.click()
+        jobCreatePage.tab(JobTab.OTHER).click()
+        jobCreatePage.defaultTabNodes.click()
+        jobCreatePage.updateBtn.click()
+        jobShowPage.runJobBtn.click()
+        def logOutputSwitch = executionShowPage.logOutputSwitch
+        def nodeViewContainer = executionShowPage.nodeFlowState
+        expect:
+        currentUrl.endsWith(option)
+        logOutputSwitch.getAttribute("style") == ""
```
Comment (ronaveva) [inline functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobTabsSpec.groovy:50]: what is this supposed to do?

**Review by ronaveva — CHANGES_REQUESTED** [review]
overall looks ok, just minor changes required. Please take a look at the comments

File: functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobTabsSpec.groovy (hunk 3 above)
Comment (DarwisNarvaezDev) [reply functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobTabsSpec.groovy:50]: this switch don't show if the job output component is rendered to a view, so its a double check.

**Review by ronaveva — APPROVED** [review]
LGTM

**Pull request merged**